components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -20,19 +20,25 @@ function createLookup() {
         .map(item => item.label)
         .concat(group.name)
     );
+
     lookup.set(name, values);
   });
 
   return lookup;
 }
 
-const VulnerabilitiesGrid = ({ resources, onUpdate }) => {
+const VulnerabilitiesGrid = ({
+  resources,
+  onUpdate,
+  initialExpandedGroups = {}
```

Comment:
what is this initialExpandedGroups for? will they not all be closed initially every time?

Reply:
That's for testing

Reply:
removed it now